jsk_fetch_robot/jsk_fetch_user.rosinstall.indigo

```diff
@@ -16,3 +16,10 @@
     local-name: jsk-ros-pkg/jsk_3rdparty
     uri: https://github.com/jsk-ros-pkg/jsk_3rdparty.git
     version: 2.1.13
+# indigo is already EOL and fetch_ros is never released.
+# we cannot use 0.8.x (for melodic) because some packages of fetch_ros cannot be built in indigo PC.
```

Comment:
Please add ‘can not build in indigo PC, because it uses OpenCV 2.4.8 but 0.8.x using OpenCV 3.2’ or something

Reply:
Thank you for your review.
I updated.